src/mixins/common-mixin.ts

```diff
@@ -76,22 +79,24 @@ export default class CommonMixin extends Vue {
 
   /**
    * Navigates to the dashboard page, optionally with a filing ID.
-   * @param identifier The identifier to include in the dashboard URL
-   * @param filingId The filing ID to include in the dashboard URL (optional, defaults to null)
+   * @param identifier the identifier to include in the dashboard URL (optional)
+   * @param filingId the filing ID to include in the dashboard URL (optional)
    */
```

Comment:
The main change, below, is that I swapped the "then" and "else" conditions to avoid a double negative condition (ie, "if !isNoRedirect").

I also provide a default value for identifier and a proper "no value" default for filing id (NaN).